sidecars/go/Dockerfile

```diff
@@ -39,18 +39,16 @@ RUN set -e -x && \
     go get -u -v github.com/uudashr/gopkgs/v2/cmd/gopkgs && \
     go get -u -v golang.org/x/tools/cmd/gotype && \
     go get -u -v github.com/mdempsky/gocode && \
-    go get -u -v github.com/stamblerre/gocode && \
-    GO111MODULE=on go get -v golang.org/x/tools/gopls@master golang.org/x/tools@master && \
-    go build -o /go/bin/gocode-gomod github.com/stamblerre/gocode && \
+    go get -v golang.org/x/tools/gopls@master golang.org/x/tools@master && \
+    go get -v honnef.co/go/tools/cmd/staticcheck@master && \
```

Comment:
Hello, Sorry let me know if it makes sense. I am not a go-lang expert. Is this line mandatory? I am just wondering because the licence looks strange https://github.com/dominikh/go-tools/blob/master/LICENSE

Reply:
It's MIT license, i think, so what is the problem ?

Reply:
@disaster37 I don't see any reference to the MIT license or did I miss something ?

Reply:
Create new github repository, initialized with MIT license. You can look it's the same contend.

Reply:
I think we are good enough to merge it. Looks like it is MIT license and we are including this deps on a docker image